pkg/vulnsrc/vulnerability/const.go

```diff
@@ -47,4 +48,5 @@ const (
 	Swift     types.Ecosystem = "swift"
 	Cocoapods types.Ecosystem = "cocoapods"
 	Bitnami   types.Ecosystem = "bitnami"
+	K8s       types.Ecosystem = "k8s"
```

Comment:
It's recommended to use the same value.

```suggestion
	K8s       types.Ecosystem = "kubernetes"
```